Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -1402,7 +1404,7 @@ void Tileset::_unloadCachedTiles() noexcept {
     const TileRenderContent* pRenderContent =
         pTile->getContent().getRenderContent();
     if (_options.enableLodTransitionPeriod && pRenderContent &&
-        pRenderContent->getLodTransitionFadePercentage() > 0.0f) {
+        pRenderContent->getLodTransitionFadePercentage() < 1.0f) {
```

Comment:
... but tiles with a fade less than 1.0 will not be unloaded.

So... tiles will pretty much never be unloaded, right?